src/test/java/com/github/awwkoala/selenium/DropDownTests.java

```diff
@@ -0,0 +1,34 @@
+package com.github.awwkoala.selenium;
+
+import com.github.awwkoala.selenium.page.DropDownPage;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class DropDownTests extends TestBase {
+
+  private DropDownPage ddPage;
+/*  Stwórz test automatyczny na podstawie poniższego scenariusza testowego:
+  Przejdź na stronę dropdown
+  Sprawdź, że po wejściu na stronę w kontrolce typu Dropdown jest wybrana opcja "Please select an option"
+  Ustaw wartość na Option 1
+  Sprawdź, że wybrana opcja to Option 1
+  Ustaw wartość na Option 2
+  Sprawdź, że wybrana opcja to Option 2*/
+
+  @BeforeClass
+  public void prepareTest() {
+    ddPage = new DropDownPage(driver);
+    ddPage.goTo();
+  }
+
+  @Test
+  public void dropDownTest() {
```

Comment:
It's my personal opinion but I don't like naming test names like dropdownTest, xxxTest. You are in test class, and one line above is @Test annotation, too many "Test" redundancy. Think about how you can describe it.It's good practice to name it, so  other people don't need to read the whole test body. What about `shouldVerifyOptionsFromDropdown` or sth like that. Or even `verifyAllOptionsFromDropdown`